openstf/model/metamodels/missing_values_handler.py

```diff
@@ -0,0 +1,120 @@
+# SPDX-FileCopyrightText: 2017-2021 Alliander N.V. <korte.termijn.prognoses@alliander.com> # noqa E501>
```

Comment:
I noticed an __init__.py is missing in the metamodels folder. Could you add it?

Reply:
done.